Apps/Sandcastle/gallery/Web Map Tile Service with Time.html

```diff
@@ -0,0 +1,95 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, minimum-scale=1, user-scalable=no">
+    <meta name="description" content="Use Viewer to start building new applications or easily embed Cesium into existing applications.">
+    <meta name="cesium-sandcastle-labels" content="Beginner, Showcases">
+    <title>Cesium Demo</title>
+    <script type="text/javascript" src="../Sandcastle-header.js"></script>
+    <script type="text/javascript" src="../../../ThirdParty/requirejs-2.1.20/require.js"></script>
+    <script type="text/javascript">
+    require.config({
+        baseUrl : '../../../Source',
+        waitSeconds : 60
+    });
+    </script>
+</head>
+<body class="sandcastle-loading" data-sandcastle-bucket="bucket-requirejs.html">
+<style>
+    @import url(../templates/bucket.css);
+</style>
+<div id="cesiumContainer" class="fullSize"></div>
+<div id="loadingOverlay"><h1>Loading...</h1></div>
+<div id="toolbar"></div>
+<script id="cesium_sandcastle_script">
+function startup(Cesium) {
+    'use strict';
+//Sandcastle_Begin
```

Comment:
No. This server has missing tiles. If you look at the Capabilities file you see

```xml
      <TileMatrix>
        <ows:Identifier>0</ows:Identifier>
        <ScaleDenominator>223632905.6114871</ScaleDenominator>
        <TopLeftCorner>-180 90</TopLeftCorner>
        <TileWidth>512</TileWidth>
        <TileHeight>512</TileHeight>
        <MatrixWidth>2</MatrixWidth>
        <MatrixHeight>1</MatrixHeight>
      </TileMatrix>
      <TileMatrix>
        <ows:Identifier>1</ows:Identifier>
        <ScaleDenominator>111816452.8057436</ScaleDenominator>
        <TopLeftCorner>-180 90</TopLeftCorner>
        <TileWidth>512</TileWidth>
        <TileHeight>512</TileHeight>
        <MatrixWidth>3</MatrixWidth>
        <MatrixHeight>2</MatrixHeight>
      </TileMatrix>
      <TileMatrix>
        <ows:Identifier>2</ows:Identifier>
        <ScaleDenominator>55908226.40287178</ScaleDenominator>
        <TopLeftCorner>-180 90</TopLeftCorner>
        <TileWidth>512</TileWidth>
        <TileHeight>512</TileHeight>
        <MatrixWidth>5</MatrixWidth>
        <MatrixHeight>3</MatrixHeight>
      </TileMatrix>
      <TileMatrix>
        <ows:Identifier>3</ows:Identifier>
        <ScaleDenominator>27954113.20143589</ScaleDenominator>
        <TopLeftCorner>-180 90</TopLeftCorner>
        <TileWidth>512</TileWidth>
        <TileHeight>512</TileHeight>
        <MatrixWidth>10</MatrixWidth>
        <MatrixHeight>5</MatrixHeight>
      </TileMatrix>
```

Level 0 has the correct 2 columns, 1 row, so Level 1 should have 4 columns and 2 rows but instead has 3 columns and 2 rows. You'll see that the errors you get at Level 2 and 3 don't exist either. It's always been a problem with WMTS, but its server dependent. So we would probably need to make another tiling scheme or allow WebMercator to parse this and handle accordingly. Regardless its out of the scope if this PR.